packages/utilities/gel-foundations/src/typo_fonts.js

```diff
@@ -0,0 +1,193 @@
+const fontFamilyBase = ', Helvetica, Arial, sans-serif';
+const baseFontUrl = 'https://gel.files.bbci.co.uk/r2.302/';
+
+export const FF_SERIF = `ReithSerif${fontFamilyBase}`;
+export const FF_SANS = `ReithSans${fontFamilyBase}`;
+export const FF_SANS_COND = `ReithSansCondensed${fontFamilyBase}`;
+
+// Serif
+export const T_REITH_SERIF_REGULAR = `
```

Comment:
Thanks for compiling this all into one file. It looks great. 

I think these font-face declarations should live within the `psammead-styles` utility package, since there's a difference between what GEL provide us and what our UX team & we have decided should be the usage of the fonts with our particular font-faces.  Here, we've got choices with regards to which `font-weight` and `font-display` to use. We might decide to change it. Whereas everything within `gel-foundations` should be only updated if and when there is a change to the GEL specification for typography.

Reply:
Updated to reflect 👍